test/base/BaseModel.spec.js

```diff
@@ -4,10 +4,39 @@ import _isString from 'lodash/isString';
 import BaseModel from '../../src/assets/scripts/client/base/BaseModel';
 import ExtendedBaseModelFixture from './_fixtures/ExtendedBaseModelFixture';
 
+ava('sets the #_id when passed valid parameters', (t) => {
+    const modelInt = new BaseModel();
+    t.true(modelInt._id.indexOf('BaseModel') !== -1);
+
+    const modelBool = new BaseModel('string');
+    t.true(modelBool._id.indexOf('string') !== -1);
+    console.log(modelBool._id);
+});
+
+ava('makes sure the model fails to instantiate with any incorrect', t => {
```

Comment:
take a look at some of the other test files and try to stay close to the convention established there with how this type of test is worded.  its usually along the lines of: `throws when passed invalid parameters` or something like that.